src/web/crate_details.rs

```diff
@@ -149,7 +166,10 @@ impl CrateDetails {
 
             versions.sort();
             versions.reverse();
-            versions.iter().map(|semver| format!("{}", semver)).collect()
+            versions.iter()
+                .map(|semver| format!("{}", semver))
+                .map(|version| map_to_release(&conn, crate_id, version))
```

Comment:
I think there only needs to be one `.map()` here.

```suggestion
                .map(|version| map_to_release(&conn, crate_id, version.to_string()))
```

Reply:
Yes, indeed. I was bit confused by the use of `format!("{}", semver)` instead of simply `semver.to_string()`...